docker/package/tezos_setup_wizard.py

```diff
@@ -66,6 +67,7 @@
 secret_key_regex = b"(encrypted|unencrypted):(?:\w{54}|\w{88})"
 address_regex = b"tz[123]\w{33}"
 derivation_path_regex = b"(?:bip25519|ed25519|secp256k1|P-256)\/[0-9]+h\/[0-9]+h"
+signer_uri_regex = b"((?:tcp|unix|https|http):\/\/.+)\/keys\/(tz[123]\w{33})\/?"
```

Comment:
Can we import it from `wizard_structure` to avoid duplication?

Reply:
Good catch, missed it 👍